internal/routes.go

```diff
@@ -0,0 +1,19 @@
+package internal
+
+import "github.com/julienschmidt/httprouter"
+
+type Route struct {
+	Name        string
+	Method      string
+	Path        string
+	HandlerFunc httprouter.Handle
+}
+
+func Routes() []Route {
+	return []Route{
+		{"Index", "GET", "/", Index},
```

Comment:
I don't know if we will use the Name for something later, but if not, we should be adding things that we don't need, or we are not sure if we will use it.